mlflow/environment_variables.py

```diff
@@ -740,6 +740,15 @@ def get(self):
 #: (default: ``False``)
 MLFLOW_ENABLE_UC_FUNCTIONS = _BooleanEnvironmentVariable("MLFLOW_ENABLE_UC_FUNCTIONS", False)
 
+#: Enable multi-turn evaluation support in mlflow.genai.evaluate.
+#: When enabled, multi-turn scorers can evaluate entire conversation sessions by grouping
+#: traces with the same session_id. Multi-turn assessments are logged to the first trace
+#: in each session chronologically.
+#: (default: ``False``)
+MLFLOW_ENABLE_MULTI_TURN_EVALUATION = _BooleanEnvironmentVariable(
```

Comment:
Curious why do we decide to use env var to control this behavior instead of adding a parameter in `genai.evaluate` function?